packages/date-picker/src/DatePickerInput.tsx

```diff
@@ -53,6 +53,7 @@ export const DateInput = forwardRef<HTMLInputElement, DateInputProps>(
 			<Field
 				label={label}
 				secondaryLabel="(dd/mm/yyyy)"
+				hideOptionalLabel={false}
```

Comment:
Why is this hardcoded?

Reply:
The `DatePicker` component uses the `secondaryLabel` prop to render "(dd/mm/yyyy)" which means that components actually never renders "(optional)". 

We don't need to explicitly set `hideOptionalLabel` so I've just removed it for now,.